docs/references/http-websocket-apis/public-api-methods/subscription-methods/subscribe.md

```diff
@@ -547,14 +548,150 @@ Example order book stream message:
+    },
+    {
+      "currency_a": "XRP_drops",
+      "currency_b": "rchGBxcD1A1C2tdxF6papQYZ8kjRKMYcL/BTC",
+      "volume_a": "352373535",
+      "volume_b": "0.00249291478138912",
+      "high": "1413500174054660e-4",
+      "low": "1413499999999996e-4",
+      "open": "1413500174054660e-4",
+      "close": "1413499999999996e-4"
+    },
+    {
+      "currency_a": "XRP_drops",
+      "currency_b": "rcvxE9PS9YBwxtGg1qNeewV6ZB3wGubZq/5553445400000000000000000000000000000000",
+      "volume_a": "8768045",
+      "volume_b": "4.193604075536",
+      "high": "2090813.734932601",
+      "low": "2090813.734932601",
+      "open": "2090813.734932601",
+      "close": "2090813.734932601"
+    },
+    {
+      "currency_a": "XRP_drops",
+      "currency_b": "rhub8VRN55s94qWKDv6jmDy1pUykJzF3wq/USD",
+      "volume_a": "28113",
+      "volume_b": "0.013405652999",
+      "high": "2097100.380123005",
+      "low": "2097100.380123005",
+      "open": "2097100.380123005",
+      "close": "2097100.380123005"
+    },
+    {
+      "currency_a": "r3dVizzUAS3U29WKaaSALqkieytA2LCoRe/58434F5245000000000000000000000000000000",
+      "currency_b": "rcoreNywaoz2ZCQ8Lg2EbSLnGuRBmun6D/434F524500000000000000000000000000000000",
+      "volume_a": "75.626516003375",
+      "volume_b": "63.022096669479",
+      "high": "1.200000000000003",
+      "low": "1.200000000000003",
+      "open": "1.200000000000003",
+      "close": "1.200000000000003"
+    },
+    {
+      "currency_a": "rKiCet8SdvWxPXnAgYarFUXMh1zCPz432Y/CNY",
+      "currency_b": "rKiCet8SdvWxPXnAgYarFUXMh1zCPz432Y/USD",
+      "volume_a": "83.9115222024",
+      "volume_b": "11.51049687275",
+      "high": "7.290000000004561",
+      "low": "7.290000000004561",
+      "open": "7.290000000004561",
+      "close": "7.290000000004561"
+    },
+    {
+      "currency_a": "rcRzGWq6Ng3jeYhqnmM4zcWcUh69hrQ8V/LTC",
+      "currency_b": "rchGBxcD1A1C2tdxF6papQYZ8kjRKMYcL/BTC",
+      "volume_a": "0.64167647147626",
+      "volume_b": "0.00073047551165797",
+      "high": "878.4366638381051",
+      "low": "878.4366638381051",
+      "open": "878.4366638381051",
+      "close": "878.4366638381051"
+    },
+    {
+      "currency_a": "rhub8VRN55s94qWKDv6jmDy1pUykJzF3wq/USD",
+      "currency_b": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B/USD",
+      "volume_a": "0.013432464305",
+      "volume_b": "0.013566788948",
+      "high": "0.9900990099046391",
+      "low": "0.9900990099046391",
+      "open": "0.9900990099046391",
+      "close": "0.9900990099046391"
+    }
+  ]
+}
+```
 
-{% badge href="https://github.com/XRPLF/rippled/releases/tag/1.4.0" %}New in: rippled 1.4.0{% /badge %}
+The fields from a Book Changes stream message are as follows:
+
+| Field          | Value            | Description                             |
+|:---------------|:-----------------|:----------------------------------------|
+| `type`         | String           | The value `bookChanges` this is from the Book Changes stream. |
```

Comment:
I can't parse the description. Perhaps "The value `bookChanges` indicates this is from the Book Changes stream."